app/views/notifications/_nav_menu.html.erb

```diff
@@ -4,24 +4,24 @@
       <h3 class="crayons-subtitle-3 mb-4 pl-2"><%= @user.name %></h3>
     <% end %>
     <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].blank? %>" href="/notifications">
-      All
+      <%= t("core.all") %>
     </a>
     <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].blank? %>" href="/notifications/comments">
-      Comments
+      <%= t("core.comment").pluralize %>
```

Comment:
@benhalpern We really shouldn't have used `pluralize` here. Rails I18n has this built-in:

https://guides.rubyonrails.org/i18n.html#pluralization